SQL/SQL (basic).sublime-syntax

```diff
@@ -0,0 +1,751 @@
+%YAML 1.2
+---
+name: SQL (Basic)
+scope: source.sql.basic
+hidden: true
+version: 2
+
+variables:
+  string_escape: (?:\\.)
+  simple_identifier: (?:\w+)
+  reserved: (?i:;|\b(?:from|order|group|select|where|inner|outer|left|right|join|on|set|union|insert|delete|update|truncate|create|alter|drop|return)\b)
+  additional_reserved: (?!)
+  simple_types: (?i:\b(?:bit|bool|boolean|datetime|int)\b)
```

Comment:
Are there reasons for this extra grouping here?

Would the following be enough?

```suggestion
  simple_types: \b(?i:bit|bool|boolean|datetime|int)\b
```

Reply:
It would, but generally I thought we try to make a variable a group https://github.com/sublimehq/Packages/issues/757#issuecomment-544136278

Reply:
Reasonable. 

I've never really paid attention to it, tbh. Only had a discussion about `(?x) ...` vs. `(?x: ... )` in mind. Other than that I've always have used a style suitable for applied use cases. The atomic style is more bullet prove though. Maybe worth keeping it this way especially with complexity of inheritance in mind.